### DISCUSSION TITLE
Device Request - Lumia 928

### DISCUSSION THREAD

**Discussion by Maxxas234** [RedGreenBlue09/WFAv7_Installer#47] [General]
Any chance you could add support for the Lumia 928? Seems like similar chipsets are supported so hoping it's doable. Thanks.

**Comment by RedGreenBlue09** [comment]
I don't think it is possible or it is very hard. There is no engineering SBL3 for 928 which means it cannot get to mass storage mode.

**Comment by RedGreenBlue09** [comment]
>Therefore you should only use an alternative SBL3 if it came from a phone that is at least the same brand and CPU architecture as your phone.

Perhaps use the engineering SBL3 of 925 and drivers of 920. It will not work as good like on the 920 though.

Also, no warranty for doing this.

**Reply by Maxxas234** [reply]
Thanks, that's what I was concerned about after not being able to get into Mass Storage mode in Winternals.

**Reply by RedGreenBlue09** [reply]
Is this really working?

**Reply by Maxxas234** [reply]
Are you asking if WP Internals is working?  The 928 is visible from the app i.e. can see General Info, Phone Identity and Security details, as well as boot it into Flash mode.  It fails when I try to boot into Mass Storage mode though.

**Reply by RedGreenBlue09** [reply]
Did you tried unlocking it with engineering SBL3 of 925? That might help enabling Mass Storage Mode. No pressure of course.

**Reply by Maxxas234** [reply]
Was finally able to unlock the bootloader but it continuously boot loops whenever I try to put it into Mass Storage mode. Had to restore using Recovery Tool. 

Oh well - was worth a try.

**Reply by RedGreenBlue09** [reply]
It is actually possible to do by getting the GPT and partitions using thor2, but it is tricky to mount and do operations on them from a batch script. Additionally, writing the large partitions back may harm the eMMC.

**Comment by RedGreenBlue09** [comment]
Hi, can you use WPinternals backup feature?
